tox.ini

```diff
@@ -36,7 +36,7 @@ allowlist_externals =
     sh
 commands =
     {toxinidir}/ci/build_ndcore.sh {toxinidir}/neurodamus/data {toxworkdir}
-    sh -c ". {toxworkdir}/.envfile && pytest -x --cov=neurodamus --cov-report=term-missing tests/integration"
+    sh -c ". {toxworkdir}/.envfile && pytest -x --cov=neurodamus --cov-report=term-missing --forked tests/integration"
```

Comment:
I had to run the tests in this folder in separate forked processes, otherwise failure in CI. Instead of setting`pytestmark = pytest.mark.forked` in each test file,  I would prefer to set it centrally during launching.